node/cn/api_tracer.go

```diff
@@ -553,8 +555,14 @@ func (api *PrivateDebugAPI) traceBlock(ctx context.Context, block *types.Block,
 // be one filename per transaction traced.
 func (api *PrivateDebugAPI) standardTraceBlockToFile(ctx context.Context, block *types.Block, config *StdTraceConfig) ([]string, error) {
 	// If we're tracing a single transaction, make sure it's present
-	if config != nil && config.TxHash != (common.Hash{}) {
-		if !containsTx(block, config.TxHash) {
+	if config != nil && !common.EmptyHash(config.TxHash) {
+		var exists bool
+		for _, tx := range block.Transactions() {
+			if exists = (tx.Hash() == config.TxHash); exists {
+				break
+			}
+		}
+		if !exists {
```

Comment:
We just changed this code from https://github.com/klaytn/klaytn/commit/05749a25def8159d697dc6e5d0158c3c45638e8c
@yoomee1313 @ehnuje Which one is the latest one?

Reply:
@aidan-kwon @ehnuje's commit is latest. I removed it https://github.com/klaytn/klaytn/pull/761/commits/d1ab996946a763bf85c922a9f5436a47c656bf1c.